libraries/AP_DDS/README.md

```diff
@@ -33,20 +33,25 @@ Currently, serial is the only supported transport, but there are plans to add IP
 
 While DDS support in Ardupilot is mostly through git submodules, another tool needs to be available on your system: Micro XRCE DDS Gen.
 
-1. Go to a directory on your system to clone the repo (perhaps next to `ardupilot`)
-1. Install java
+- Go to a directory on your system to clone the repo (perhaps next to `ardupilot`)
+- Install java
   ```console
-  sudo apt install java
+  sudo apt install default-jre
   ````
-1. Follow instructions [here](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-micro-xrce-dds-gen-tool) to install the generator, but use `develop` branch instead of `master` (for now).
+- Follow instructions [here](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-micro-xrce-dds-gen-tool) to install the generator, but use `develop` branch instead of `master` (for now).
   ```console
   git clone -b develop --recurse-submodules https://github.com/eProsima/Micro-XRCE-DDS-Gen.git
   cd Micro-XRCE-DDS-Gen
   ./gradlew assemble
   ```
 
-1. Add the generator directory to $PATH, like [so](https://github.com/eProsima/Micro-XRCE-DDS-docs/issues/83). 
-1. Test it
+- Add the generator directory to $PATH, like [so](https://github.com/eProsima/Micro-XRCE-DDS-docs/issues/83). 
```

Comment:
Feel free to remove the link now that you have the content placed here

Reply:
okk!